_docs/20b-typography.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "Typography"
+permalink: /customize/typography/
+excerpt: ""
+---
+
+{% include base_path %}
+{% include toc %}
+
+## What is typography?
+Typography is a way of arranging type to make text legible, readable, and appealing when displayed.
+
+## Custom Typography of Ontimize Web framework.
+
+To change the typography it is neccesary to apply an extension of Angular Material's (you can see spec in [Angular Material's typography](https://v6.material.angular.io/guide/typography){:target='_blank'}) and an extension of Ontimize Web Sass-based theming. 
+
+*The extension of Ontimize Web Sass-based theming.* is arranged into typography levels, as Angular Material's typography. Each level has a font-size, line-height and font-weight. This custom typography is used to apply css about [o-table]({{ base_path }}/components/table/){:target="_blank"} depending of the `row-height` attribute. For more details about table typography and defaut config, see [the source](https://github.com/OntimizeWeb/ontimize-web-ngx-theming/blob/master/src/styles/lite/typography/o-table-typography.scss){:target='_blank'}. The available levels are:
+<ul>
+  <li><strong>small-header-height</strong>: table header height for 'small' predefined row height options.</li>
+  <li><strong>small-row-height</strong>: table row height for 'small' predefined row height options. </li>
+  <li><strong>small-header-font-size</strong> : table header font size  for 'small' predefined row height options. </li>
+  <li><strong>small-row-font-size</strong>:  table row font size for 'small' predefined row height options </li>
+
+<li><strong>medium-header-height</strong>:  table header height for 'medium' predefined row height options</li>
+<li><strong>medium-row-height</strong>: table row height for 'medium' predefined row height options.</li>
+<li><strong>medium-header-font-size</strong>:  table header font size for 'medium' predefined row height options </li>
+<li><strong>medium-row-font-size</strong>: table row font size for 'medium' predefined row height options</li>
+<li><strong>large-header-height</strong>:  table header height  for 'large' predefined row height options</li>
+<li><strong>large-row-height</strong>: table row height for 'large' predefined row height options.</li>
+<li><strong>large-header-font-size</strong>: table header font size for 'large' predefined row height options</li>
+<li><strong>large-row-font-size</strong>:  table row font size for 'large' predefined row height options </li>
+</ul>
+
+## Customization
+
+First you must create a custom *typography configuration* of Angular Material's Sass-based theming and another *typography configuration* of Ontimize Web Sass-based theming as the below example demonstrates
```

Comment:
The paragraph starts with `First ...` but there is only one step. Start with `Create a custom ...`

Reply:
There are three steps, define typography configuration of angular Material and then define typography configuration of ontimize web and at end merge both